tests/core/training/story_reader/test_yaml_story_reader.py

```diff
@@ -0,0 +1,56 @@
+from rasa.core import training
```

Comment:
should we also test the raised warnings?

Reply:
Frankly speaking, I'm not a big fan of those test we have that check for an exact string match from the stdout/stderr.
Do you think that would be enough to check that we have non-empty output?

Reply:
> Frankly speaking, I'm not a big fan of those test we have that check for an exact string match from the stdout/stderr.

I strongly agree!  I'd just use `pytest.warns(...)` to check that a warning was raised. The message itself isn't important.

Reply:
I'll keep the "schema" warning for the schema. Added two cases:
- intent with no leading slash
- wrong yaml file